pyqtgraph/functions.py

```diff
@@ -1179,10 +1179,9 @@ def imageToArray(img, copy=False, transpose=True):
             # If this works on all platforms, then there is no need to use np.asarray..
             arr = np.frombuffer(ptr, np.ubyte, img.byteCount())
     
+    arr = arr.reshape(img.height(), img.width(), 4)
```

Comment:
It's a bugfix--as the name implies, `RGB32` still uses 4 bytes per pixel, even though the alpha channel is unused. I guess I had only used this with `ARGB32` in the past.